debugger/src/main/kotlin/org/rust/debugger/runconfig/RsLocalDebugProcess.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.debugger.runconfig
+
+import com.intellij.execution.filters.TextConsoleBuilder
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.util.io.FileUtil
+import com.intellij.openapi.util.text.StringUtil
+import com.intellij.xdebugger.XDebugSession
+import com.jetbrains.cidr.execution.RunParameters
+import com.jetbrains.cidr.execution.debugger.CidrLocalDebugProcess
+import com.jetbrains.cidr.execution.debugger.backend.DebuggerCommandException
+import com.jetbrains.cidr.execution.debugger.backend.gdb.GDBDriver
+import com.jetbrains.cidr.execution.debugger.backend.lldb.LLDBDriver
+
+class RsLocalDebugProcess(
+    parameters: RunParameters,
+    debugSession: XDebugSession,
+    consoleBuilder: TextConsoleBuilder
+) : CidrLocalDebugProcess(parameters, debugSession, consoleBuilder) {
+
+    fun loadPrettyPrinters(sysroot: String) {
+        val threadId = currentThreadId
+        val frameIndex = currentFrameIndex
+        postCommand { driver ->
+            when (driver) {
+                is LLDBDriver -> driver.loadPrettyPrinters(threadId, frameIndex, sysroot)
+                is GDBDriver -> driver.loadPrettyPrinters(threadId, frameIndex, sysroot)
+            }
+        }
+    }
+
+    private fun LLDBDriver.loadPrettyPrinters(threadId: Long, frameIndex: Int, sysroot: String) {
+        val path = "$sysroot/lib/rustlib/etc/lldb_rust_formatters.py".systemDependentAndEscaped()
+        try {
+            executeConsoleCommand(threadId, frameIndex, """command script import "$path"""")
+            executeConsoleCommand(threadId, frameIndex, """type summary add --no-value --python-function lldb_rust_formatters.print_val -x ".*" --category Rust""")
+            executeConsoleCommand(threadId, frameIndex, """type category enable Rust""")
+        } catch (e: DebuggerCommandException) {
```

Comment:
LGTM for now, but make sure that CLion folks are aware that we load pretty printers this way. They might add a proper API for this in the future hopefully!

Reply:
Currently, there isn't special API for pretty printers loading and gdb pretty-printers are loaded in the same way